tests/test_simpleButler.py

```diff
@@ -169,8 +169,6 @@ def testImportTwice(self):
         """Test exporting dimension records and datasets from a repo and then
         importing them all back in again twice.
         """
-        if self.datasetsIdType is not uuid.UUID:
```

Comment:
I think there is one more test `if self.datasetsIdType is int` above which we should probably drop. Maybe eliminate `self.datasetsIdType` completely?